credentialdigger/snippet_similarity.py

```diff
@@ -53,38 +53,41 @@ def compute_snippet_embedding(snippet, model):
 
     Returns
     -------
-    np.array
+    list
         The 128 element embedding for the input snippet
     """
 
-    """ Compute snippet's token embeddings """
-    small_bert_result = tf.squeeze(model(tf.constant([snippet])))
+    # Preprocess snippet
+    preprocessed_snippet = snippet.replace("\'", "\"")
+    # Compute snippet's token embeddings
+    small_bert_result = tf.squeeze(model(tf.constant([preprocessed_snippet])))
     small_bert_embeddings = small_bert_result.numpy()
-    """ Compute snippet's embedding as the mean of
-    the token embeddings """
+    # Compute snippet's embedding as mean of token embeddings
     snippet_embedding = np.mean(small_bert_embeddings, axis=0)
-    return snippet_embedding
+    return snippet_embedding.tolist()
 
 
 def compute_similarity(emb_1, emb_2):

```

Comment:
please use better variables names (e.g., embedding_1 and 2), otherwise we risk not to understand it in 6 months :D